payment-request/idlharness.https.window.js

```diff
@@ -10,16 +10,21 @@ idl_test(
   ['dom', 'html'],
   idlArray => {
     try {
-      const methods = [{supportedMethods: 'foo'}];
+      const methods = [
+        {supportedMethods: 'basic-card'},
+        {supportedMethods: 'https://apple.com/apple-pay'},
+      ];
       const amount = {currency: 'USD', value: '0'};
-      const details = {total: {label: 'bar', amount: amount} };
+      const details = {total: {label: 'label', amount: amount} };
       window.paymentRequest = new PaymentRequest(methods, details);
     } catch (e) {
-      // Will be surfaced in idlharness.js's test_object below.
+      // Surfaced below when paymentRequest is undefined.
     }
 
     idlArray.add_objects({
       PaymentRequest: ['paymentRequest'],
+      PaymentMethodChangeEvent: ['new PaymentMethodChangeEvent("type")'],
```

Comment:
I guess the "type" doesn't really matter, right?

Reply:
(if it does, "paymentmethodchange" would be the right type here)

Reply:
I don't think it would cause an issues, but doesn't hurt to use real values.